node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/LoadBalancerProvisioner.java

```diff
@@ -345,12 +349,12 @@ private Set<String> reachableIpAddresses(Node node) {
     private static void requireInstance(LoadBalancerId id, LoadBalancer loadBalancer, CloudAccount cloudAccount, ZoneEndpoint zoneEndpoint) {
         if (loadBalancer.instance().isEmpty()) {
             // Signal that load balancer is not ready yet
-            throw new LoadBalancerServiceException("Could not (re)configure " + id + ". The operation will be retried on next deployment");
+            throw new LoadBalancerServiceException("Could not provision " + id + ". The operation will be retried on next deployment");
         }
-        if (!inAccount(cloudAccount, loadBalancer)) {
+        if ( ! inAccount(cloudAccount, loadBalancer)) {
```

Comment:
Hopefully we'll decide on (and enforce) a code style some day. 🙂

Reply:
🙈Yes.